dev-python/exam/exam-0.10.6.ebuild

```diff
@@ -1,9 +1,9 @@
-# Copyright 1999-2020 Gentoo Authors
```

Comment:
For the future: if you see nose, check if the package in question actually uses it (i.e. `import`s anything from nose). If not, try to swap it for unittest or pytest. (I'll do it here)

Reply:
Thanks for the note - will check in the future.